src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java

```diff
@@ -24,9 +24,18 @@
 
 public class WindowsCommandLineArgsSanitizer {
 
-    private static final char[] SPECIAL_CHARS = "|<>&^\r\n".toCharArray();
+    private static final char[] SPECIAL_CHARS = " |<>&^\r\n".toCharArray();
```

Comment:
No, it breaks in certain scenarios like a winrs -r command run from a localhost as winrs proxy. This one is a specific case.